packages/forms-web-app/src/routes/full-appeal/date-decision-due.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const dateDecisionDueController = require('../../controllers/full-appeal/date-decision-due');
+const fetchExistingAppealMiddleware = require('../../middleware/fetch-existing-appeal');
+const combineDateInputsMiddleware = require('../../middleware/combine-date-inputs');
+const { validationErrorHandler } = require('../../validators/validation-error-handler');
+const {
+  rules: decisionDateDueValidationRules,
+} = require('../../validators/full-appeal/date-decision-due');
+
+const router = express.Router();
+
+router.get(
+  '/date-decision-due',
+  [fetchExistingAppealMiddleware],
+  dateDecisionDueController.getDateDecisionDue
+);
+
+router.post(
+  '/date-decision-due',
+  [fetchExistingAppealMiddleware, combineDateInputsMiddleware],
+  decisionDateDueValidationRules('decision-date', 'the date the decision was due'),
```

Comment:
No we need to specify the message from here?

Reply:
Moved it to validator.